src/libraries/pkg/baseline/packageIndex.json

```diff
@@ -1551,6 +1545,13 @@
       "BaselineVersion": "5.0.1",
       "InboxOn": {}
     },
+    "runtime.osx-x64.runtime.native.System.IO.Ports": {
```

Comment:
@Anipik are these changes necessary or should we just update it for the platforms package?

Reply:
its fine we can keep these, it will help to reduce the noise in future.